packages/near-fast-auth-signer-e2e-tests/pages/DevicesPage.ts

```diff
@@ -0,0 +1,37 @@
+import { Page } from '@playwright/test';
+
+import { checkConditionUntilMet } from '../utils/async';
+
+class DevicesPage {
+  private page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async getCount() {
+    const checkboxes = this.page.locator('input[type="checkbox"]');
+    return checkboxes.count();
+  }
+
+  async selectAndDelete() {
+    const checkFunction = async () => {
+      const count = await this.getCount();
+      return count > 0;
+    };
+    // Wait until it loads records and it is available for an action
+    await checkConditionUntilMet(this.page, checkFunction, 5, 1000);
+    const checkboxes = this.page.locator('input[type="checkbox"]');
+    const count = await checkboxes.count();
+
+    for (let i = 0; i < count; i += 1) {
+      const element = checkboxes.nth(i);
+      /* eslint-disable no-await-in-loop */
```

Comment:
Not a big deal for me but maybe this can run a bit faster? Didn't tested, but not a blocker for me too. 

```
await Promise.all(
      Array.from({ length: count }, (_, i) => checkboxes.nth(i).click())
    );
```

Reply:
this was actually my original implementation. Just to prove my point, I actually replace my code with yours and it was erroring out like below:
<img width="277" alt="image" src="https://github.com/near/fast-auth-signer/assets/6027014/a3733d42-0473-4fa1-b894-e70591e6d39d">
(sometimes it only selects just one)

One that I learn from using loop + select action is that it needs to be run sequentially to guarantee that it gets clicked properly in playwright.

Reply:
So let's leave like you implemented, no issues for me.